pkg/ingester/retention.go

```diff
@@ -289,12 +289,19 @@ func (dc *diskCleaner) CleanupBlocksWhenHighDiskUtilization(ctx context.Context)
 }
 
 // isBlockDeletable returns true if this block can be deleted.
-func (dc *diskCleaner) isBlockDeletable(block *tenantBlock) bool {
+func (dc *diskCleaner) isBlockDeletable(block *tenantBlock, deleteOnlyUploaded bool) bool {
```

Comment:
Super minor suggestion: what do you think about splitting the function?
```go
func (dc *diskCleaner) isExpired(block *tenantBlock) bool
```

Caller can check `block.Unloaded` on its own, we do not have to hide it in the function

Reply:
Make sense, Updating PR

Reply:
Updated PR, splitting function makes code more readable now. Thanks for reviewing